### DISCUSSION TITLE
How to use CLI version? (windows 10) [command window instantly closing]

### DISCUSSION THREAD

**Discussion by AlexiosTheSixth** [Tyrrrz/DiscordChatExporter#1127] [Q&A]
Recently decided to try to switch to the CLI version as the GUI version does not support exporting threads.

Downloaded it and tried opening the exe but it keeps "crashing" (the commandline window instantly closes), am I doing anything wrong? I got it to open properly ONCE but it instantly crashed the second I pressed a key.

I just downloaded the latest cli version zip file from releases, am I doing anything wrong? Am I missing some sort of dependency? I tried looking if there was a .bat file I needed to run to install dependencies but all there was was the exe, dll libraries, configs and a json, and some pdb libraries.

**Comment by 96-LB** [comment]
You need to open a command prompt window and run the program as a command. Read [the wiki](https://github.com/Tyrrrz/DiscordChatExporter/blob/master/.docs/Getting-started.md#using-the-cli) for more information.

**Comment by AlexiosTheSixth** [comment]
Thanks 👍
